darkdraw/drawing.py

```diff
@@ -957,6 +968,8 @@ def select_top(sheet, box):
 Drawing.addCommand('r', 'reset-time', 'sheet.autoplay_frames.extend([[0, f] for f in sheet.frames])')
 Drawing.addCommand('c', 'set-default-color', 'vd.default_color=list(itercursor())[-1].color')
 
+Drawing.addCommand('q', 'stop-animation', 'sheet.stop_animation()', 'stop animation and return to frame 0')
```

Comment:
`q` quits the sheet and is a pretty fundamental VisiData command, so we can't override it for this.  How about a `Ctrl+` keystroke?  What do durdraw and other drawing tools use for this?

Reply:
`Alt+P` is fine for this.  We can add a binding to it to toggle animation if that makes it easier.